setup.py

```diff
@@ -71,6 +71,7 @@
         "Programming Language :: Python :: 3.8",
     ],
     packages=find_packages(),
+    package_data={"pyscal": ["py.typed"]},
```

Comment:
I think you would also need to create a file `py.typed`.

See e.g. https://github.com/equinor/webviz-config/blob/master/webviz_config/py.typed

Reply:
👍🏻